### DISCUSSION TITLE
Merchant Shop

### DISCUSSION THREAD

**Discussion by lstebner** [jeremyckahn/farmhand#101] [Ideas]
In addition to the normal store, what if we had a "traveling merchant" type shop that showed up randomly. it would only be available for a day at a time when it shows up, and could offer things for sale like higher level seeds the player hasn't unlocked yet, or fully grown produce at cheapish prices (maybe priced with the intent to use for recipes rather than just for reselling). this idea is probably quite a ways out in terms of when it would make sense to implement, but i wanted to write it down while i was thinking about it

**Comment by jeremyckahn** [comment]
Strong +1 to this idea! I think it would be a great addition. 🙂
